modules/standard/Set.chpl

```diff
@@ -569,9 +581,8 @@ module Set {
     :arg b: A set to take the union of.
 
     :return: A new set containing the union between `a` and `b`.
-    :rtype: `set(?t, ?)`
   */
-  proc +(const ref a: set(?t, ?), const ref b: set(t, ?)): set(t, ?) {
```

Comment:
What's the difference between the return type of `set(t, ?)` here and `set(t)` above?  (I think nothing other than coding style?).

Reply:
D'oh!  I'm reading the red code... nevermind!